### ISSUE TITLE
mAP is zero while training on COCO Dataset for Persons

### ISSUE THREAD

**Issue by adithya-p** [NVIDIA/DIGITS#1977]
I am training DetectNet on COCO Dataset for **persons** only.  I set parameters as follows:

```
custom classes: dontcare, person
solver type: ADAM
subtract mean: none
learning rate: 0.0001
epochs: 200
policy: Exponential Decay
gamma: 0.99
```

I got the network from here: [detectnet_network.prototxt](https://github.com/NVIDIA/caffe/blob/caffe-0.15/examples/kitti/detectnet_network.prototxt) My mAP is zero even after 100 epochs. I have checked similar issues and responses from @lukeyeager @gheinrich @dusty-nv  and tried them, but they didn't solve my issue. Can you give a detailed insight to fix this issue?

**Comment by mahxn0** [comment]
i also have this problem @lukeyeager @gheinrich @dusty-nv

**Comment by adithya-p** [comment]
@mahxn0 setting crop_bboxes: false and increase number of epochs. If you are not currently using exponential learning rate decay I recommend trying that with a decay rate of 0.99. This worked for me.
